go.mod

```
@@ -1,3 +1,5 @@
 module example.com/hello
```

We should probably update this module to point to your repository
```suggestion
module github.com/bmasaiati/go-projects
```